plugins/woocommerce-admin/client/two-column-tasks/task-list.tsx

```diff
@@ -299,30 +272,14 @@ export const TaskList: React.FC< TaskListProps > = ( {
 					<List animation="custom">
 						{ visibleTasks.map( ( task, index ) => {
 							++index;
-							const className = classnames(
-								'woocommerce-task-list__item index-' + index,
-								{
-									'is-complete': task.isComplete,
-									'is-active': task.id === activeTaskId,
-								}
-							);
+
 							return (
-								<TaskItem
+								<TaskListItem
```

Comment:
Making use of `TaskListItem` here actually breaks this task list design, it's suppose to look like this:
<img width="812" alt="Screen Shot 2022-04-22 at 12 17 23 PM" src="https://user-images.githubusercontent.com/2240960/164744132-7fcd1d09-cb20-4af7-a068-d715de1284f0.png">
But now looks like:
<img width="795" alt="Screen Shot 2022-04-22 at 12 18 39 PM" src="https://user-images.githubusercontent.com/2240960/164744295-f07f682e-48cb-4aa9-93da-f408576d0bd5.png">